Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -325,6 +325,13 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate, Loa
         animateSearchEnginesWithKeyboard(state)
     }
 
+    override func viewWillTransitionToSize(size: CGSize, withTransitionCoordinator coordinator: UIViewControllerTransitionCoordinator) {
+        // The height of the suggestions row may change, so call reloadData() to recalculate cell heights.
+        coordinator.animateAlongsideTransition({ _ in
+            self.tableView.reloadData()
```

Comment:
Is it safe to call `reloadData()` here? On what thread will it run? Will it run during an animation, possibly interfering with its framerate? Would `systemLayoutFittingSizeDidChangeForChildContentContainer` be better?